src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStep.java

```diff
@@ -261,35 +287,11 @@ public boolean getShowGitToolOptions() {
             return false;
         }
 
-        @Inject
-        public void setJenkinsToBitbucketCredentials(
-                JenkinsToBitbucketCredentials jenkinsToBitbucketCredentials) {
-            this.jenkinsToBitbucketCredentials = jenkinsToBitbucketCredentials;
-        }
-
-        private BitbucketMirrorHandler createMirrorHandler(BitbucketScmHelper helper) {
-            injectJenkinsToBitbucketCredentials();
-            return new BitbucketMirrorHandler(
-                    bitbucketClientFactoryProvider,
-                    jenkinsToBitbucketCredentials,
-                    (client, project, repo) -> helper.getRepository(project, repo));
-        }
-
-        BitbucketScmHelper getBitbucketScmHelper(String bitbucketUrl,
-                                                 @Nullable BitbucketTokenCredentials tokenCredentials) {
-            return new BitbucketScmHelper(bitbucketUrl,
-                    bitbucketClientFactoryProvider,
-                    jenkinsToBitbucketCredentials.toBitbucketCredentials(tokenCredentials));
-        }
-
-        private Optional<BitbucketServerConfiguration> getConfiguration(@Nullable String serverId) {
-            return bitbucketPluginConfiguration.getServerById(serverId);
-        }
-
-        private void injectJenkinsToBitbucketCredentials() {
-            if (jenkinsToBitbucketCredentials == null) {
-                Guice.createInjector(new JenkinsToBitbucketCredentialsModule()).injectMembers(this);
-            }
+        @VisibleForTesting
+        List<BitbucketServerConfiguration> getConfigurationByName(String serverName) {
+             return bitbucketPluginConfiguration.getServerList()
```

Comment:
No need to change, but I wonder would it be valuable to move this to the `BitbucketPluginConfiguration` instead so other classes can also use `getByName` ?